packages/KaSim/KaSim.4.0.0/opam

```diff
@@ -34,11 +34,9 @@ build: [
   [make "check"] {with-test}
 ]
 install: [ [ make "install-lib" ] ]
-remove: [ [ "ocamlfind" "remove" "KappaLib" ] ]
 
-synopsis: "Software suite for the Kappa language."
-flags: light-uninstall
+synopsis: "Software suite for the Kappa language"
 url {
-  src: "https://github.com/Kappa-Dev/KaSim/archive/v4.0.tar.gz"
+  src: "https://opam.ocaml.org/cache/md5/89/894a517c7991f42100d2e957efc6cfa2"
   checksum: "md5=894a517c7991f42100d2e957efc6cfa2"
```

Comment:
```suggestion
  src: "https://github.com/Kappa-Dev/KappaTools/archive/v4.0.tar.gz"
  checksum: "md5=2f9ee76a8aa8bf2ad6bbb37071759e59"
```
`diff -r` does not show any difference between the two archives apart from their top directory name